internal/pkg/istio/config_factory.go

```diff
@@ -222,97 +228,120 @@ func (cf *ConfigFactory) EnvoyFilters() []*v1alpha3.EnvoyFilter {
 }
 
 func (cf *ConfigFactory) ServiceEntries() ([]*v1alpha3.ServiceEntry, error) {
-	remote := cf.cfg.MeshPeers.Remote
 
-	if len(remote.Addresses) == 0 {
-		return nil, nil
-	}
+	var serviceEntries []*v1alpha3.ServiceEntry
+	serviceEntriesByHost := make(map[string]*v1alpha3.ServiceEntry, len(cf.cfg.MeshPeers.Remotes))
```

Comment:
I don't understand the purpose of this map. You never add entries to this map.

Reply:
Yeah, noticed it this morning 
... Silly partial git add or some other brain fart... Must have been tired or stupid. Most likely both xD

Reply:
This has been fixed with the latest changes pushed just now. PTAL at [this comment](https://github.com/openshift-service-mesh/federation/pull/110/files#diff-c1818d5fcdc653f21b85a94acbfa2fdafb66928e2610bcfb8d9cc32001572caeR287-R288) where I explained my line of thinking